src/meta/greedy_load_balancer.h

```diff
@@ -39,26 +39,19 @@
 
 namespace dsn {
 namespace replication {
-enum class cluster_balance_type
+enum class balance_type
 {
     COPY_PRIMARY = 0,
     COPY_SECONDARY,
+    MOVE_PRIMARY,
     INVALID,
 };
-ENUM_BEGIN(cluster_balance_type, cluster_balance_type::INVALID)
-ENUM_REG(cluster_balance_type::COPY_PRIMARY)
-ENUM_REG(cluster_balance_type::COPY_SECONDARY)
-ENUM_END(cluster_balance_type)
-
-// TODO: combine with cluster_balance_type
-enum class balance_type
-{
-    move_primary,
-    copy_primary,
-    copy_secondary
-};
+ENUM_BEGIN(balance_type, balance_type::INVALID)
+ENUM_REG(balance_type::COPY_PRIMARY)
+ENUM_REG(balance_type::COPY_SECONDARY)
```

Comment:
It seems that there are `ENUM_REG(balance_type::MOVE_PRIMARY)` missing.

Reply:
Thanks for reminding me. fixed.